.github/workflows/build.yml

```diff
@@ -34,11 +34,11 @@ jobs:
       run: sudo gem install cocoapods
 
     - name: CocoaPods spec lint
-      run: pod lib lint Google-Maps-iOS-Utils.podspec
+      run: pod spec lint Google-Maps-iOS-Utils.podspec
```

Comment:
this shouldn't be changed, `lib lint` validates the local sources, `spec lint` validates a published  tag, since this changes are not published you can't properly test